sdk/storage/azure-storage-file-datalake/azure/storage/file/datalake/_data_lake_lease.py

```diff
@@ -0,0 +1,245 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+import uuid
+
+from typing import (  # pylint: disable=unused-import
+    Union, Optional, Any,
+    TypeVar, TYPE_CHECKING
+)
+from azure.storage.blob import LeaseClient as BlobLeaseClient
```

Comment:
`LeaseClient` has been renamed to `BlobLeaseClient` so you can just do: `from azure.storage.blob import BlobLeaseClient`

Reply:
Thanks for reminding me ^_^, the base branch is not very updated, I will do this after rebase!